src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -136,6 +118,22 @@ class HoldRequest extends React.Component {
   }
 
   /**
+   * modelDeliveryLocationName(prefLabel, shortName)
+   * Renders the names of the radio input fields of delivery locations except EDD.
+   *
+   * @param {String} prefLabel
+   * @param {String} shortName
+   * @return {String}
+   */
+  modelDeliveryLocationName(prefLabel, shortName) {
+    if (prefLabel && typeof prefLabel === 'string' && shortName) {
+      return `${shortName} - ${prefLabel.split(' - ')[1]}`;
```

Comment:
No it is not correct.... I accidentally updated the changes on `dev-eb-deploy` branch, so I had to reverse some changes back. I think this one is a part I missed to correct.